v1/src/main/java/com/google/cloud/teleport/options/WindowedFilenamePolicyOptions.java

```diff
@@ -96,10 +96,10 @@ public interface WindowedFilenamePolicyOptions extends PipelineOptions {
       regexes = {"^[^A-Za-z0-9/]?(d+|D+)[^A-Za-z0-9/]?$"},
       description = "Custom Day Pattern to use for the output directory",
       helpText =
-          "Pattern for formatting the day. Must be one or more of 'd' for day of month or 'D' for"
-              + " day of year. Case makes no difference in the year. The pattern can be optionally"
-              + " wrapped by characters that aren't either alphanumeric or the directory ('/')"
-              + " character. Defaults to 'dd'")
+          "Pattern for formatting the day. Must be one or more of `d` for day of month or `D` for"
+              + " day of year. Case makes no difference in the year. Optionally,"
```

Comment:
Is it true that "Case makes no difference in the year."? It seems like lowercase if for day of the month and uppercase if for day of the year, so the case should make a difference.

Reply:
@damccorm please confirm.

Reply:
Lets cut this line, it just doesn't make sense here. This parameter has no bearing on the year at all, it looks like this was a bad copy paste from https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/64443e50f64add3b2f923f457d24b67aea331e03/v1/src/main/java/com/google/cloud/teleport/options/WindowedFilenamePolicyOptions.java#L73